packages/driver/src/cy/testConfigOverrides.ts

```diff
@@ -52,6 +52,22 @@ function setConfig (testConfig: ResolvedTestConfigOverride, config, localConfigO
 
       try {
         testConfig.applied = overrideLevel
+        // this is unique validation, not applied to the general cy config.
```

Comment:
I am surprised this logic isn't pushed up the [`validateOverridableAtRunTime`](https://github.com/cypress-io/cypress/blob/feature/experimental-retries/packages/config/src/browser.ts#L191C14-L191C42) method to ensure the stack & messaging is consistent with the existing override error messaging (found [here](https://github.com/cypress-io/cypress/blob/develop/packages/driver/src/util/config.ts#L97)). That is the place that ensure these levels are appropriate for the config passed in.

Reply:
We can't declare invalid override levels for these keys because these are subkeys, and not defined as a primary config key. I can move the subkey detection to `validateOverridableAtRunTime`, but this logic will be removed in the next few weeks when we add support for setting experimental retry config at test & suite level overrides.